jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/OperationsEvent.java

```diff
@@ -52,7 +50,7 @@ public class OperationsEvent {
 
   @Enumerated(EnumType.STRING)
   @Column(name = "publisher_role", nullable = false)
-  private PublisherRole publisherRole;
+  private PartyFunction publisherRole;
```

Comment:
PartyFunction seem to define a lot of values that are not valid for this field

Reply:
I'm not Pim, but I think it would make sense to only include the enums specific to JIT since this is a JIT-specific implementation and a `PublisherRole` for an OperationsEvent in JIT only has the nine Finn mentioned.

Reply:
I follow your argument and will change it (y)